src/server.js

```diff
@@ -70,13 +70,44 @@ app.post('/api/npsResponses/create', promiseHandler(async req => {
   if (errors) {
     throw new Response(400, errors)
   }
+  console.log('Creating NPS response for: ' + email)
   await datastore.save({
     key: datastore.key('NpsResponse'),
     data: { ...data, email, timestamp: new Date() }
   })
   return new Response(201)
 }))
 
+/**
+ * @api {post} /api/firstTimestamps/record Record & return timestamp when user first accessed app
+ * @apiName firstVisitTimestamp
+ * @apiVersion 1.0.0
+ * @apiGroup Visits
+ * @apiSuccess (Success 200) timestamp Timestamp when user first accessed app
+ */
+app.post('/api/firstTimestamps/record', promiseHandler(async req => {
+  const email = await validateUser(req)
+
+  const query = datastore.createQuery('FirstTimestamp')
+    .filter('email', email)
+    .order('timestamp')
+    .limit(1)
+  const [entities] = await datastore.runQuery(query)
+
+  if (entities.length) {
+    console.log(email + ' first visited on ' + entities[0].timestamp)
+    return new Response(200, entities[0])
```

Comment:
This should follow the same format as the other case, e.g. return an object with just `timestamp`.